install-manager/src/main/java/cm/aptoide/pt/install_manager/workers/PackageDownloader.kt

```diff
@@ -37,5 +37,8 @@ interface PackageDownloader {
    * @param packageName - a package name
    * @returns true there was something to cancel and it was cancelled.
    */
-  fun cancel(packageName: String): Boolean
```

Comment:
Let's not do this. It wasn't meant to be used like this. The implementation itself must keep track of all the files and not rely on the input here.